focuspoints.lrdevplugin/DefaultPointRenderer.lua

```diff
@@ -164,3 +201,36 @@ function DefaultPointRenderer.createPointView(x, y, rotation, iconFileTemplate,
 
   return view
 end
+
+--[[
+-- Takes a LrPhoto and returns the rotation and horizontal mirroring that the use has set in Lightroom (generaly in grid mode)
+-- photo - the LrPhoto to calculate the values from
+-- returns:
+-- - rotation in rad in trigonometric sense
+-- - horizontal mirroring (0 -> none, -1 -> yes)
+--]]
+function DefaultPointRenderer.getLightroomRotationAndMirroring(photo)
+  local lrRotation = photo:getRawMetadata("orientation")
```

Comment:
is appears this is a LR 6 feature only. Anyway we can branch the code to check what version of LR is running and just not support this in LR5?